src/js/components/sharedComponents/CanopyDensityPicker.tsx

```diff
@@ -27,12 +32,29 @@ const CanopyDensityPicker = (props: CanopyDensityPickerProps): JSX.Element => {
   return (
     <>
       <div>
-        {props.label && <span>Change canopy density</span>}
-        <button
-          className="canopy-density-picker"
-          style={{ backgroundColor: `${customColorTheme}` }}
-          onClick={handleDensityButtonClick}
-        >{`> ${markValueMap[density]}%`}</button>
+        {props.label &&
```

Comment:
This component is too complex. Why are we checking for props.label existence? I see from the types that this type is required, do we have a scenario that props.label does not exist?

Also this is just two spans and a button. I do not believe we need this map function here with index checking and so on.
```
<span>{displayLabel[0]}</span>
<button></button>
<span>{displayLabel[1]}</span>
```
would that work?

Reply:
yep - I'll implement that now. I noticed `props.label` was used to conditionally render static text, so I included it just in case,  but I'll also remove that logic and update the props interface, etc for `CanopyDensityPicker`